vscode/src/toitExec.ts

```diff
@@ -41,11 +41,8 @@ async function executeRunCommand(ctx: Context, device?: Device) {
   try {
     if (!device) device = await selectDevice(ctx, { "activeOnly": true, "simulatorOnly": false });
 
-    const commandProcess = cp.spawn(ctx.toitExec, [ "dev", "-d", device.name, "run", filePath ]);
-    const toitOutput: OutputChannel = ctx.outputChannel(device.deviceID, device.name);
-    toitOutput.show(true);
-    commandProcess.stdout.on("data", data => toitOutput.append(`${data}`));
-    commandProcess.stderr.on("data", data => toitOutput.append(`${data}`));
+    cp.spawn(ctx.toitExec, [ "dev", "-d", device.name, "run", filePath ]);
```

Comment:
How is the output of the command piped into the output view?

Edit: it seems like we are falling back to `logs`.

Reply:
Correct.